common/djangoapps/student/views.py

```diff
@@ -1754,6 +1755,9 @@ def create_account_with_params(request, params):
             }
         )
 
+    # Announce registration
+    REGISTER_USER.send(sender=None, user=user, profile=profile)
```

Comment:
It seems safer to me to have an explicit signal for new registration.  To use post_save, I would have to decide whether to use the User or Profile post_save which would make assumptions about the order they are saved.